src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomColor = new Random().nextInt(Colors.values().length);
```

Comment:
You're creating a new `Random` object each time `getRandomColor()` is called. It would be more efficient to create a single `Random` instance as a class field and reuse it.